kratos/solving_strategies/builder_and_solvers/residualbased_block_builder_and_solver.h

```diff
@@ -1308,21 +1305,11 @@ class ResidualBasedBlockBuilderAndSolver
                 #pragma omp for schedule(guided, 512) nowait
                 for (int i_const = 0; i_const < static_cast<int>(rModelPart.MasterSlaveConstraints().size()); ++i_const) {
                     auto it_const = it_const_begin + i_const;
+                    it_const->EquationIdVector(slave_ids, master_ids, r_current_process_info);
 

```

Comment:
why are u removing this???

Reply:
green means adding : P , what i am actually removing is the `if(constraint_is_active)` as I am adding the indices from all constraints, not only active ones.